src/Paramore.Brighter.MessagingGateway.AzureServiceBus/AzureServiceBusProducerRegistryFactory.cs

```diff
@@ -39,12 +39,12 @@ public class AzureServiceBusProducerRegistryFactory : IAmAProducerRegistryFactor
         /// Creates message producers.
         /// </summary>
         /// <returns>A has of middleware clients by topic, for sending messages to the middleware</returns>
-        public IAmAProducerRegistry Create()
+        public IAmAProducerRegistry Create(int bulkSendBatchSize = 10)
```

Comment:
Can you please move this to the Constructor and plumb it in as a private readonly variable